xml/src/org/netbeans/modules/xml/resources/templates/emptyDTD.dtd.template

```diff
@@ -1,4 +1,24 @@
 <?xml version="1.0"?>
+<#--
+
+    Licensed to the Apache Software Foundation (ASF) under one
+    or more contributor license agreements.  See the NOTICE file
+    distributed with this work for additional information
+    regarding copyright ownership.  The ASF licenses this file
+    to you under the Apache License, Version 2.0 (the
+    "License"); you may not use this file except in compliance
+    with the License.  You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing,
+    software distributed under the License is distributed on an
+    "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+    KIND, either express or implied.  See the License for the
+    specific language governing permissions and limitations
+    under the License.
+
+-->
```

Comment:
It seems that the license header is visible for users. (Tools > Templates > XML > DTD Entitty : Opne in Editor)
![apache-nb-module-review-xml-templates](https://user-images.githubusercontent.com/738383/31313624-08ab3982-ac23-11e7-949d-b8bdaec2da57.png)